.github/workflows/ci-full.yml

```
@@ -0,0 +1,2790 @@
+        run: pnpm start engines download mongodb 8.0
+
+      - name: Show installed engines
+        run: pnpm start engines list
+
+      - name: Run FerretDB integration tests
+        run: pnpm test:engine ferretdb
+        timeout-minutes: 20
+
+      - name: Show FerretDB logs on failure (Unix)
+        if: failure() && runner.os != 'Windows'
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2010:warning:5:1: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)